_monitoring-your-cluster/metrics/getting-started.md

```diff
@@ -0,0 +1,123 @@
+
+
+## Get started
+
+The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag. It's important to exercise caution when enabling this feature because it can consume system resources. Detailed information on enabling and configuring metrics framework is described in the following sections.
+
+### Enabling the flag on a node using tarball
+
+The enable flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
+
+#### Option 1: Enable the experimental feature flag in the `opensearch.yml` file
+
+1. Change to the top directory of your OpenSearch installation:
+
+```bash
+cd \path\to\opensearch
+```
+
+2. Open your OpenSearch configuration folder, and then open the `opensearch.yml` file with a text editor.
+3. Add the following line:
+
+```bash
+opensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+4. Save your changes and close the file.
+
+#### Option 2: Modify jvm.options
+
+Add the following lines to `config/jvm.options` before starting the OpenSearch process to enable the feature and its dependency:
+
+```bash
+-Dopensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+Run OpenSearch:
+
+```bash
+./bin/opensearch
+```
+{% include copy.html %}
+
+#### Option 3: Enable from an environment variable
+
+As an alternative to directly modifying `config/jvm.options`, you can define the properties by using an environment variable. You can enable this feature in a single command when you start OpenSearch or by setting an environment variable.
+
+To add these flags inline when starting OpenSearch, run the following command:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true" ./opensearch-2.9.0/bin/opensearch
+```
+{% include copy.html %}
+
+To define the environment variable separately, prior to running OpenSearch, run the following command:
+
+```bash
+export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+ ./bin/opensearch
+```
+{% include copy.html %}
+
+### Enable with Docker containers
+
+If you’re running Docker, add the following line to `docker-compose.yml` under `environment`:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+```
+{% include copy.html %}
+
+### Enable for OpenSearch development
```

Comment:
I wouldn't recommend it, instead, I would keep this in the GitHub repo with the Metrics Framework source code.